pkg/server/db_options.go

```diff
@@ -711,6 +728,18 @@ func (opts *dbOptions) Validate() error {
 		)
 	}
 
+	if opts.RetentionPeriod < 0 || (opts.RetentionPeriod > 0 && opts.RetentionPeriod < Milliseconds(store.MinimumRetentionPeriod.Milliseconds())) {
+		return fmt.Errorf(
+			"%w: invalid retention period for database '%s'. RetentionPeriod should at least 1 day",
+			ErrIllegalArguments, opts.Database)
+	}
+
+	if opts.TruncationFrequency < 0 || (opts.TruncationFrequency > 0 && opts.TruncationFrequency < Milliseconds(store.MinimumTruncationFrequency.Milliseconds())) {
+		return fmt.Errorf(
+			"%w: invalid truncation frequency for database '%s'. TruncationFrequency should at least 1 hour",
```

Comment:
it would be better use the constant value denoting the minimum value in the error message

Reply:
noted